tests/Unit/inc/classes/subscriber/third-party/plugins/Smush_Subscriber/maybeDeactivateRocketLazyload.php

```diff
@@ -0,0 +1,136 @@
+<?php
+namespace WP_Rocket\Tests\Unit\inc\classes\third_party\plugins\Smush_Subscriber;
+
+use Brain\Monkey\Functions;
+
+/**
+ * @covers \WP_Rocket\Subscriber\Third_Party\Plugins\Smush_Subscriber::maybe_deactivate_rocket_lazyload
+ * @group ThirdParty
+ * @group Smush
+ */
+class Test_MaybeDeactivateRocketLazyload extends SmushSubscriberTestCase {
+
+	public function testShouldNotDisableWPRocketLazyLoad() {
+		Functions\expect( 'update_rocket_option' )->never();
+
+		// Smush not enabled, WPR enabled.
+		$this->mock_is_smush_lazyload_enabled(
+			false,
+			[]
+		);
+
+		Functions\when( 'get_rocket_option' )
+			->alias(
+				function( $option ) {
+					switch ( $option ) {
+						case 'lazyload':
+						case 'lazyload_iframes':
+							return true;
+					}
+					return false;
+				}
+			);
```

Comment:
This won't be needed anymore as we are removing `get_rocket_option` and injecting the option instance instead.